frenchchocolatine.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "Frenchchocolatine",
+  "logoUrl": "https://imagizer.imageshack.com/v2/320x240q70/922/gyMFuT.jpg",
```

Comment:
Should be updated with the logo submitted in your other PR which should be merged first.
You can look at my PR for example of how to use in the logoURL the GitHub location.

Reply:
it's better but we should still simplify it as in the documentation by having consistent naming and path, that's the purpose of this repository too.